l10n/en/firefox/new/desktop.ftl

```diff
@@ -29,7 +29,7 @@ firefox-desktop-download-compare-edge = { -brand-name-edge }
 firefox-desktop-download-compare-safari = { -brand-name-safari }
 firefox-desktop-download-yes = Yes
 firefox-desktop-download-no = No
-firefox-desktop-download-blocks-third-party = Blocks third-party tracking cookies
+firefox-desktop-download-blocks-third-party = Blocks third-party tracking cookies by default
```

Comment:
I think this needs to be a new string with a new ID. That also means we can set the old one as the fallback.